.github/workflows/storybook.yml

```diff
@@ -53,10 +54,11 @@ jobs:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
-    if: github.event_name != 'pull_request'
+    if: github.repository == 'EddieHubCommunity/LinkFree' && github.event_name != 'pull_request'
     needs: build
```

Comment:
Hello @poorvijn, I initially suggested adding the conditional check in #6772 without realizing that the `deploy` steps are dependent on the `build` steps due to the `needs: build` condition. Therefore, the additional conditional check `github.repository == 'EddieHubCommunity/LinkFree'` might not be necessary here. If the `build` steps are skipped, the `deploy` steps should automatically be skipped as well. These are just my thoughts. Feel free to give it a try. 😄

Reply:
surely going to check it out !